frontend/src/data/routes.js

```diff
@@ -0,0 +1,75 @@
+import React, {useContext} from 'react';
+import { Route, Redirect } from "react-router-dom";
+
+import Info from '../pages/info';
+import Questionnaire from '../pages/questionnaire';
+import Login from '../pages/login';
+import dashboard from '../pages/dashboard';
+import NewUser from '../pages/newUser';
+import Prequestionnaire from '../pages/prequestionnaire';
+
+import {AuthContext} from './authContext'
+
+
+export const routes = [
+    {
+      path: '/login',
+      label: 'Login',
+      component: Login
+    },
+    {
+      path: '/',
+      label: 'Info',
+      component: Info
+    },
+    {
+      path: '/questionnaire',
+      label: 'Questionnaire',
+      component: Questionnaire
+    },
+    {
+      path: '/newuser',
+      label: 'New User',
+      component: NewUser
+    },
+    {
+      path: '/prequestionnaire',
+      label: 'Prequestionnaire',
+      component: Prequestionnaire
+    }
```

Comment:
I think that the /application route is specified elsewhere in App.js but that means there are two separate ways to configure routes. What if you configured all routes here with an optional "requiresLogin" property?